bench/workgen/workgen.cxx

```diff
@@ -2086,6 +2166,22 @@ int WorkloadRunner::run_all() {
         thread_handles.push_back(thandle);
     }
 
+    // Start Timestamp increment thread
+    if (options->oldest_timestamp_lag > 0 || options->stable_timestamp_lag > 0) {
+
```

Comment:
I think before starting (probably at the beginning of `WorkloadRunner::run`), we should check to make sure that `timestamp_advance` is > 0 if either of `oldest_timestamp_lag` and `stable_timestamp_lag` is set.  Throw an exception if not.

Reply:
Fixed